src/components/PrivacyStatement/style.css

```diff
@@ -0,0 +1,84 @@
+@import (less) "../Theme/constants.css";
+
+.privacy {
+  height: calc(~"100% - 112px");
```

Comment:
`calc` isn't a less mixin, or anything - it's a native CSS function

You should just be able to do `calc(100% - 112px`), right?

Reply:
what do we need calc for?

Reply:
We need it because the privacy component has an absolute positioning and we want to make sure that its height never covers the footer so it has a 100% height minus the height (including paddings/margins) of the footer.
@stephencookdev If I use `calc(100% - 112px)` it gets computed as `height: calc(-12%);`

Reply:
so we can't use padding instead? Also, the computation is a bit weird, does padding have a height os 12%?

Reply:
No, `calc` does not work as expected with LESS that's why I'm using the following syntax `calc(~"100% - 112px");` see [this issue](https://github.com/less/less.js/issues/974).
I cannot use a padding but I could get rid of absolute positioning and give it a fixed height

Reply:
I added a comment for now as this behaviour is also used in the confirm screen where there is already a comment explaining why we are using `calc`.  This will have to be refactored as part of the header/footer positioning ticket